modules/network_synchronizer/net_utilities.h

```diff
@@ -199,7 +196,8 @@ T StatisticalRingBuffer<T>::average() const {
 	}
 	a = a / T(data.size());
 	T b = avg_sum / T(data.size());
-	ERR_FAIL_COND_V_MSG(ABS(a - b) > (CMP_EPSILON * 4.0), b, "The `avg_sum` accumulated a sensible precision loss: " + rtos(ABS(a - b)));
+	const T difference = a > b ? a - b : b - a;
+	ERR_FAIL_COND_V_MSG(difference > (CMP_EPSILON * 4.0), b, "The `avg_sum` accumulated a sensible precision loss: " + rtos(a - b));
```

Comment:
`rtos(a - b)` ups, you forgot to change this too

Reply:
Oops :)